pyuvdata/tests/test_uvcal.py

```diff
@@ -1311,3 +1333,24 @@ def test_add_errors(self):
         """Test behavior that will raise errors"""
         # test addition of two identical objects
         nt.assert_raises(ValueError, self.delay_object.__add__, self.delay_object2)
+
+    def test_multi_files(self):
```

Comment:
Same as for above, needs a slightly more detailed story about whats happening here.